tests/sentry/sentry_metrics/test_gen_metrics_multiprocess_steps.py

```diff
@@ -66,6 +66,7 @@ def compare_messages_ignoring_mapping_metadata(actual: Message, expected: Messag
     actual_deserialized = json.loads(actual_payload.value)
     expected_deserialized = json.loads(expected_payload.value)
     del actual_deserialized["mapping_meta"]
+    actual_deserialized.pop("aggregation_option", None)
```

Comment:
In local testing, the metrics payload adds the aggregation option to the transformed payload. Others don't. Since the tests are generic over different payload types, removed the checking of field which is specific to a payload.

Reply:
Can you add a comment to here?